uaclient/tests/test_util.py

```diff
@@ -468,3 +468,31 @@ def test_replace_top_level_logger_name(self, logger_name, new_logger_name):
         assert (
             util.replace_top_level_logger_name(logger_name) == new_logger_name
         )
+
+
+class TestSetFilenameExtension:
+    @pytest.mark.parametrize(
+        "input_string,extension,output_string",
```

Comment:
you could add a test for a file with multiple dots, like "some.file.txt"

Reply:
done